webapp/src/components/RadioField/RadioField.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+type Props = {
+    heading: {
+        id: string;
+        message: string;
+    };
+    divClass?: string;
+    labelClass?: string;
+    options: {
+        value: string;
+        checked: boolean;
+        id?: string;
+        message?: string;
+    }[];
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    description: {
+        id: string;
+        message: string;
+    }
+}
+
+export const RadioField = ({heading, divClass, labelClass, options, onChange, description}: Props) => {
+    return (
+        <>
+            <div className='form-group'>
+                <label className='col-sm-4'>
+                    <FormattedMessage
+                        id={heading.id}
+                        defaultMessage={heading.message}
+                    />
+                </label>
+                <div className='col-sm-8'>
+                    <>
+                        <div className={`${divClass}`}>
+                            {options.map((option) => (
+                                <>
+                                    <label className={`pt-0 ${labelClass}`}>
+                                        <input
+                                            key={option.value}
+                                            type='radio'
+                                            value={option.value}
+                                            checked={option.checked}
+                                            onChange={onChange}
+                                        />
+                                        {option.value === 'true' || option.value === 'false' ?
```

Comment:
remove this condition and add translation to "true" and "false" also.....I guess it was missed by me.